bin/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "exocom-publish",
+  "version": "0.0.1",
+  "description": "Publishes the Exosphere comminication services",
+  "devDependencies": {
+    "chalk": "1.1.3",
+    "inquirer": "3.0.1",
+    "jsonfile": "2.4.0",
+    "livescript": "1.5.0",
+    "o-tools-livescript": "1.2.3",
+    "semver": "5.3.0",
+    "shelljs": "0.7.6"
+  },
+  "author": "Originate",
+  "license": "ISC"
```

Comment:
Please sort this file similar to how the other `package.json` files are sorted:
- begin with `name` and `version`
- then all other keys sorted alphabetically